projects/payment-lib/src/lib/components/fee-summary/fee-summary.component.html

```diff
@@ -123,29 +123,6 @@ <h2 class="heading-medium error-summary-heading" id="failure-error-summary-headi
         <td class="no-border"></td>
         <td class="no-border"></td>
         <td class="no-border">
-          <div class="govuk-form-group govuk-form-group--mg">
-            <fieldset class="govuk-fieldset">
-              <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
-                <span class="govuk-fieldset__heading govuk-fieldset__heading--fz">
-                  Which system are you using to take the payment?
-                </span>
-              </legend>
-              <div class="govuk-radios govuk-radios--small">
-                <div class="govuk-radios__item">
-                  <input id="changed-name" class="govuk-radios__input" [(ngModel)]="platForm" type="radio" aria-label="8x8" value="8x8">	
-                  <label class="govuk-label govuk-radios__label" for="changed-name">	
-                    8x8	
-                  </label>	
-                </div>	
-                <div class="govuk-radios__item">	
-                  <input id="changed-name-2" class="govuk-radios__input" [(ngModel)]="platForm" type="radio" aria-label="Antenna" value="Antenna">	
-                  <label class="govuk-label govuk-radios__label" for="changed-name-2">	
-                    Antenna	
```

Comment:
Antenna should be retained here. Why both the channels are removed?

Reply:
This code is to choose Antenna Or 8X8 payment before click Take Payment. As per the AC 4 , we should remove this option. When click on Take Payment by default Antenna payment should be processed.

Reply:
This code is to choose Antenna Or 8X8 payment before click Take Payment. As per the AC 4 , we should remove this radio button options. When click on Take Payment by default Antenna payment should be processed.